### PR TITLE
ts conversion in one fell swoop--hopefully

### PR DESCRIPTION
**Pull request by droddy**

### CODE REVIEW

File: src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  useState,
+  ReactElement,
+  FC,
+  SyntheticEvent,
+  Children,
+  cloneElement,
+} from 'react';
+
+interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
+  children?: any | null;
```
Comment (droddy) [inline src/components/Accordion/Accordion.tsx:11]: derp! just read the component readme

> This components takes an arbitrary amount of `ExpansionPanel` components as children

I'll go ahead and set that as the children type

File: src/components/Accordion/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  defaultExpandedPanel?: number | null;
+}
+
+const Accordion: FC<AccordionProps> = ({
+  children,
+  defaultExpandedPanel,
+  ...rest
```
Comment (pixelbandito) [inline src/components/Accordion/Accordion.tsx:18]: Minor, naming thing: The word "rest" is like writing a function that takes "arg" - no semantic benefit. That's why I prefer `passedProps` (unless you're doing something other than passing the extra props down to another node).

File: src/components/Accordion/Accordion.tsx (hunk 1 above)
Comment (pixelbandito) [inline src/components/Accordion/Accordion.tsx:11]: There's a ReactNode type, I think. I might have the name wrong.

Comment (droddy) [reply src/components/Accordion/Accordion.tsx:11]: Yeah, I was trying that but ran into other TS issues I could not resolve. After I convert children arg to the ExpansionPanel I'll try again. Thanks.

Comment (droddy) [reply src/components/Accordion/Accordion.tsx]: ReactNode was right...then had to cast to React.Element array to successfully iterate over it.

Comment (droddy) [reply src/components/Accordion/Accordion.tsx]: stuck with more generic ReactNode

**Pull request merged**